src/routes/(console)/project-[project]/auth/security/+page.svelte

```diff
@@ -11,12 +12,14 @@
 </script>
 
 <Container>
-    <UpdateUsersLimit />
-    <UpdateSessionLength />
-    <UpdateSessionsLimit />
-    <UpdatePasswordHistory />
-    <UpdatePasswordDictionary />
-    <UpdatePersonalDataCheck />
-    <UpdateSessionAlerts />
-    <UpdateMockNumbers />
+    {#if $project?.$id}
```

Comment:
cant we just rely on `$page.params.project`?

Reply:
Not in this case we need more data from the `project.authPasswordHistory` so we need to actually use the store

Reply:
I guess so, but this is only two lines of codes and future proofs all future changes

Reply:
still, the condition here potentially creates a "flicker" due to state not synced

Reply:
removed :+1: